spacy/cli/package.py

```diff
@@ -54,8 +64,8 @@ def package_cli(
         name=name,
         version=version,
         create_meta=create_meta,
-        create_sdist=create_sdist,
-        create_wheel=create_wheel,
+        create_sdist="sdist" in build and "none" not in build,
+        create_wheel="sdist" in build and "none" not in build,
```

Comment:
Should these both be the same?

Reply:
Oh no, sorry, this was a copy-paste mistake!